blues/app-accelerators
# feat: nf9 responsive design

# Problem Context

Build a responsive design of components / pages based on screen size.

## Changes

* Make the elements on the page responsive based on screen size: cards, table, font sizes, padding, displayed logo, etc.
* Flip the table to a vertical orientation where each row of data looks like a separate card with the table header displayed next to it once the table width is too large to fit comfortable on the screen.
* Convert monitor frequency values from mins (displayed in UI) to seconds (required by Notehub)

## Screenshot (if applicable)

https://user-images.githubusercontent.com/20400845/206470848-889b0797-8899-4bbb-956b-6d318d8f490d.mov

## Testing

Unit / integration / e2e tests?
 
All automated tests cont to pass and the prod build compiles.

Steps to test manually?

1. Go to http://localhost:4000
2. Resize the screen and see the elements grow and shrink accordingly. You'll notice that beyond a screen width of 1024px the elements stop growing in width - this was an intentional clamping based on the mockups which are designed to reflect mainly mobile and tablet usage.

Any other sorts of testing notes to include?

## Any other related PRs

n/a

## Ticket(s)

https://trello.com/c/NFPqfZSi/113-nf9-responsive-design-of-components-pages-based-on-screen-size
